docs/marionette.behavior.md

```diff
@@ -3,77 +3,166 @@ not be accurate or up-to-date_**
 
 # Marionette.Behavior
 
+A `Behavior` provides a clean separation of concerns to your view logic,
+allowing you to share common user-facing operations between your views.
 
-A `Behavior` is an  isolated set of DOM / user interactions that can be mixed into any `View` or another `Behavior`. Behaviors allow you to blackbox `View`-specific interactions into portable logical chunks, keeping your Views simple and your code DRY.
+Behaviors are particularly good at factoring out the common user, model and
+collection interactions to be utilized across your application.
 
 ## Documentation Index
 
-* [Motivation](#the-motivation)
-* [Using Behaviors](#using)
-* [API](#api)
-  * [Event proxy](#the-event-proxy)
-  * [Triggers](#triggers)
-  * [Model Events](#model-events)
-  * [Collection Events](#model-events)
-  * [Grouped Behaviors](#grouped-behaviors)
+* [Using Behaviors](#using-behaviors)
+  * [Defining and Attaching Behaviors](#defining-and-attaching-behaviors)
+  * [Behavior Options](#behavior-options)
+* [Nesting Behaviors](#nesting-behaviors)
+* [View Proxy](#view-proxy)
+  * [Listening to View Events](#listening-to-view-events)
+  * [Proxy Handlers](#proxy-handlers)
+  * [Template and View](#template-and-view)
   * [$](#$)
   * [$el and el](#$el-and-el)
   * [Defaults](#defaults)
   * [View](#view)
 
-## The Motivation
+## Using Behaviors
 
-As you build more and more complex Views, you will find that your `View` becomes less about displaying model data, and more about interactions.
+The easiest way to see how to use the `Behavior` class is to take an example
+view and factor out common behavior to be shared across other views.
 
-These interactions tend to be chunks of logic that you want to use in multiple views.
-
-## Usage
-
-Here is an example of a simple `View`. Let's take a stab at simplifying it, and abstracting Behaviors from it.
-
-```js
+```javascript
 var MyView = Marionette.View.extend({
```

Comment:
Would you like to use `var Mn = require('backbone.marionette');` instead if `Marionette` from global scope? In examples below `var Mn = require('backbone.marionette');` was used. So I think it'll be cool to make it same for all file)